website/docs/source/docs/boxes/base_macos.html.md

```diff
@@ -0,0 +1,160 @@
+---
+page_title: "Creating a macOS Base Box Manually"
+sidebar_current: "boxes-macos-base"
+---
+
+# Creating a macOS Base Box Manually
+
+Prior to reading this page, please check out the [basics of the Vagrant
+box file format](https://www.vagrantup.com/docs/boxes/format.html).
+
+## Pre-requisites
+
+* Install Vagrant, you can find the instructions [here](/vagrant-parallels/docs/installation)
+
+* Start Parallels Desktop and create a new macOS virtual machine as outlined  in [KB 125561](http://kb.parallels.com/125561/).
+    <div class="alert alert-info">
+    <p>
+        While creating the VM, give the username as ‘vagrant’ only.<br />
+        If this is a public box you should set the password to ‘vagrant’ as well.
+    </p>
+    </div>
+
+  * Enable passwordless sudo for macOS virtual machine
+  * Open Terminal in macOS virtual machine and run:
+
+     ```bash
+     $ sudo visudo
+     ```
+
+    Edit the line:
+
+    ```bash
+    %admin ALL=(ALL) ALL
+    ```
+
+    To Say:
+
+    ```bash
+    %admin ALL=(ALL) NOPASSWD: ALL
+    ```
+
+    Now, you should be able to run sudo without password.
+
+* Install Parallels Tools inside the macOS virtual machine.  
+    macOs 13 > **Install Parallels Tools**
+  ![install_parallels_extension](/images/install_extension.gif)
+
+      <div class="alert alert-info">
+    <p>
+        <strong>Note:</strong> Installing the Parallels Tools will require a reboot
+    </p>
+    </div>
+
+* Boot macOS virtual machine and enable Remote Login (System Settings > General > Sharing > Enable Remote Login). Don't forget to give “Full disk access to users”, do allow “All users” (in the same settings, press ![info](/images/info_32.png) button)
+![Parallels Desktop](/images/allow_sharing.gif)
+* Restart macOS virtual machine for the Remote Login to take effect.
+* Copy the vagrant public SSH key to the machine
+  
+    ```bash
+    $ ssh-copy-id -i $(find /opt/vagrant/embedded/gems -type d -name keys)/vagrant vagrant@<virtual_machine_ip>
+    ```
+
+    where:  
+
+    ```virtual_machine_ip``` \- the virtual machine ip.  
+
+## Create Base Box
+
+* In Parallels directory on your Mac create a folder (something simple like ```VagrantTest```).
```

Comment:
**nitpick:** There is no actual need to copy the VM bundle.  I believe, the fastest and easiest way is:
1) Stop the source VM  (by the way, that step should be added in any case, even if you are going to copy the bundle)
2) Unregister the source VM (`prlctl unregister <...>`)
3) Package the box (add metadata, run `tar czvf`), etc.
4) Add the box (`vagrant box add <...>`)
5) Validate the box content - make sure that the VM has `.macvm` name
6) Delete the source macvm bundle and metadata.json (rm -rf <...>), as it's not needed anymore

Benefits: 
- no need for an additional copy - it saves time and the disk space.
- no name collision, since the source VM is unregistered on the early stage.

Reply:
We will let this method stay as there are a few users that might want to maintain that VM on. we might rethink it in a later stage